features/storage/blockdevice/MBRBlockDevice.h

```diff
@@ -52,21 +52,25 @@ class MBRBlockDevice : public BlockDevice {
      *
      *  @param bd       Block device to partition
      *  @param part     Partition to use, 1-4
-     *  @param type     8-bit partition type to identify partition contents
+     *  @param type     8-bit partition type or ID to identify partition contents
```

Comment:
I don't think that adding `or ID` helps so I would leave the original text.

Reply:
```suggestion
     *  @param type     8-bit partition type or ID to identify partition contents.
     *                  Refer to https://en.wikipedia.org/wiki/Partition_type#List_of_partition_IDs 
```